app/pages/events/event_list/event_calendar.php

```diff
@@ -31,84 +31,21 @@
 $prevMonth = (clone $firstDayOfMonth)->modify('-1 month');
 $nextMonth = (clone $firstDayOfMonth)->modify('+1 month');
 
-page("Calendrier")->css("event_list.css")->noPadding()->heading(false) ?>
+page("Calendrier")->css("event_list.css")->css("event_calendar.css")->noPadding()->heading(false) ?>
 <div class="calendar-wrapper">
-    <div style="display:flex;padding-bottom:0.5rem;align-items:center;padding:0.5rem 1rem">
-        <h2 class="main-heading" style="margin:0">
+    <div class="calendar-header">
+        <h3 class="main-heading">
             <a
-                href="/evenements/calendrier?year=<?= $prevMonth->format('Y') ?>&month=<?= $prevMonth->format('m') ?>">&laquo;</a>
-            <?= $firstDayOfMonth->format('F Y') ?>
+                href="/evenements/calendrier?year=<?= $prevMonth->format('Y') ?>&month=<?= $nextMonth->format('m') ?>">&laquo;</a>
+            <span><?= $firstDayOfMonth->format('F Y') ?></span>
             <a
                 href="/evenements/calendrier?year=<?= $nextMonth->format('Y') ?>&month=<?= $nextMonth->format('m') ?>">&raquo;</a>
-        </h2>
-        <a style="margin-left:2rem" href="/evenements" role="button">
+        </h3>
+        <a style="margin-left:2rem" href="/evenements">
             <sl-tooltip content="Liste"><i class="fa fa-list"></i></sl-tooltip>
         </a>
     </div>
 
-    <style>
```

Comment:
The reason behind this inline style was that i wanted to try out another way of writing styles that didn't involve another css file.
But I think it's ok to split it in a CSS file like we always do